tests/python/gaia-ui-tests/gaiatest/tests/accessibility/music/test_a11y_music_select_album.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.music.app import Music
+from gaiatest.apps.music.regions.sublist_view import SublistView
+from marionette_driver import By
+
+
+class TestAccessibilityMusicSelectAlbum(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        # add track to storage
+        self.push_resource('MUS_0001.mp3')
+
+    def test_select_album_play(self):
+        """
+        https://moztrap.mozilla.org/mnage/case/3755/
+        """
+        music_app = Music(self.marionette)
+        music_app.launch()
+        music_app.wait_for_music_tiles_displayed()
+        self.accessibility.click(
```

Comment:
Instead of only clicking a tab we should do something similar to this (but with a11y click instead of tap):
https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/music/app.py#L60-L65
you can create a new function there and call it 'a11y_click_albums_tab'. This function will return a list view for the album view (it's definition can be found here: https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/music/regions/list_view.py).
Consequently you just need to update the list view similarly: Add a a11y_click_first_album function that is the same as https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/music/regions/list_view.py#L35-L37 but uses accessibility click instead of tap. It will return the sublist view so you can verify that it's play control locator is visible to accessibility as the last step.